pkgs/development/tools/continuous-integration/gocd-server/default.nix

```
@@ -2,12 +2,12 @@
 
 stdenv.mkDerivation rec {
   name = "gocd-server-${version}-${rev}";
-  version = "19.3.0";
-  rev = "8959";
+  version = "21.2.0";
+  rev = "12498";
```

```suggestion
  version = "21.2.0-12498";
```
then we could also convert name to pname and simplify the download URL.

---

Thanks for your feedback! I mostly left the existing code as is, which is why I decided to leave the version and name structure. While your suggested changes simplify the URL indeed, they would also require a little bit more complexity (that is, splitting the String back up again) in the build script. Is that acceptable?